services/bots/src/github-webhook/handlers/issue_comment_commands/commands.ts

```diff
@@ -0,0 +1,85 @@
+import { IssueCommentCreatedEvent } from '@octokit/webhooks-types';
+import { WebhookContext } from '../../github-webhook.model';
+import { IntegrationManifest } from '../../utils/integration';
+import { IssueCommentCommand, IssueCommentCommandContext } from './const';
+
+export const ISSUE_COMMENT_COMMANDS: { [command: string]: IssueCommentCommand } = {
+  close: {
+    description: 'Closes the issue.',
+    invokerType: 'code_owner',
+    handler: async (
+      context: WebhookContext<IssueCommentCreatedEvent>,
+      command: IssueCommentCommandContext,
+    ) => {
+      if (invokerIsCodeOwner(command)) {
+        throw new Error('Only code owners can close issues.');
+      }
+
+      await context.github.issues.update(
+        context.issue({
+          state: 'closed',
+        }),
+      );
+    },
+  },
+  rename: {
+    description: 'Change the title of the issue.',
+    exampleAdditional: 'Awesome new title',
+    invokerType: 'code_owner',
+    requireAdditional: true,
+    handler: async (
+      context: WebhookContext<IssueCommentCreatedEvent>,
+      command: IssueCommentCommandContext,
+    ) => {
+      if (invokerIsCodeOwner(command)) {
+        throw new Error('Only the issue creator can rename the issue.');
```

Comment:
Should this be code owner ?
```suggestion
        throw new Error('Only the code owner can rename the issue.');
```